src/ode_solver/low_storage_runge_kutta_ode_solver.h

```diff
@@ -0,0 +1,75 @@
+#ifndef __LOW_STORAGE_RUNGE_KUTTA_ODESOLVER__
+#define __LOW_STORAGE_RUNGE_KUTTA_ODESOLVER__
+
+#include "JFNK_solver/JFNK_solver.h"
+#include "dg/dg_base.hpp"
+#include "ode_solver_base.h"
+#include "runge_kutta_methods/low_storage_rk_tableau_base.h"
+#include "relaxation_runge_kutta/empty_RRK_base.h"
+
+namespace PHiLiP {
+namespace ODE {
+
+/// Runge-Kutta ODE solver (explicit or implicit) derived from ODESolver.
+#if PHILIP_DIM==1
+template <int dim, typename real, int n_rk_stages, typename MeshType = dealii::Triangulation<dim>>
+#else
+template <int dim, typename real, int n_rk_stages, typename MeshType = dealii::parallel::distributed::Triangulation<dim>>
+#endif
+class LowStorageRungeKuttaODESolver: public ODESolverBase <dim, real, MeshType>
+{
+public:
+    LowStorageRungeKuttaODESolver(std::shared_ptr< DGBase<dim, real, MeshType> > dg_input,
+            std::shared_ptr<LowStorageRKTableauBase<dim,real,MeshType>> rk_tableau_input,
+            std::shared_ptr<EmptyRRKBase<dim,real,MeshType>> RRK_object_input); ///< Constructor.
+
+    /// Function to evaluate solution update
+    double err_time_step(real dt, const bool pseudotime);
+
+    void step_in_time(real dt, const bool pseudotime);
+
+    /// Function to allocate the ODE system
+    void allocate_ode_system ();
+
+protected:
+    /// Stores Butcher tableau a and b, which specify the RK method
+    std::shared_ptr<LowStorageRKTableauBase<dim,real,MeshType>> butcher_tableau;
+
+    /// Stores functions related to relaxation Runge-Kutta (RRK).
+    /// Functions are empty by default.
+    std::shared_ptr<EmptyRRKBase<dim,real,MeshType>> relaxation_runge_kutta;
+
+    /// Implicit solver for diagonally-implicit RK methods, using Jacobian-free Newton-Krylov 
+    /** This is initialized for any RK method, but solution-sized vectors are 
+     *  only initialized if there is an implicit solve
+     */
+    JFNKSolver<dim,real,MeshType> solver;
```

Comment:
I think this can be removed since we will never have implicit for this type of low-storage algorithm. You can remove it from the header here and also from line 13 of the cpp file.